src/libs/actions/IOU.ts

```diff
@@ -5336,11 +5343,11 @@ function deleteMoneyRequest(transactionID: string, reportAction: OnyxTypes.Repor
                 },
             ],
             originalMessage: {
-                IOUTransactionID: undefined,
+                IOUTransactionID: null,
             },
-            errors: undefined,
+            errors: null,
         },
-    } as OnyxTypes.ReportActions;
+    } as unknown as OnyxTypes.ReportActions;
```

Comment:
NAB: Is there some way we can get around the type case to unknown? Maybe the problem is the type doesn't allow null for those fields?